### PR TITLE
docs: Prometheus Cluster Metrics

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the monitoring documentation to include mention of the Palette-specific metrics. The following updates were made.

- [Prometheus Cluster Metrics pack page](https://deploy-preview-1239--docs-spectrocloud.netlify.app/integrations/prometheus-cluster-metrics).

- Updated the guide [Enable Monitoring on Host Cluster](https://deploy-preview-1239--docs-spectrocloud.netlify.app/clusters/cluster-management/monitoring/deploy-agent).

- Updated the [Prometheus Operator pack](https://deploy-preview-1239--docs-spectrocloud.netlify.app/integrations/prometheus-operator) pack page with the new version

## Review Changes

💻 [Preview](https://deploy-preview-1239--docs-spectrocloud.netlify.app/clusters/cluster-management/monitoring/deploy-agent)

🎫 [PEM-1523](https://spectrocloud.atlassian.net/browse/PEM-1523)

### CODE REVIEW

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (amarjeet27) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:83]: 10. "... to find the Spectro Cluster Metrics pack"

and 

6. "... to find the Prometheus Agent pack"

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1239#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1239#pullrequestreview-1388991679" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:105]:
```suggestion
16. Assign a name to the host cluster and select the registered account that will deploy it.  Click on **Next**.
```

Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:105]: Or "Assign a name to the host cluster and select the registered account to deploy it."

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:108]:
```suggestion
17. Select a cluster profile to apply to your cluster. Click on **Next**.
```

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:111]:
```suggestion
18. The next screen displays all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above the cluster profile layers.
```

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:196]:
```suggestion
    - **Pack Version**: 3.3.X or newer
```

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:214]:
```suggestion
16. Assign a name to the host cluster and select the registered account that will deploy it.  Click on **Next**.
```

Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:214]: Or "Assign a name to the host cluster and select the registered account to deploy it."

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:217]:
```suggestion
17. Select a cluster profile to apply to your cluster. Click on **Next**.
```

File: content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md:220]:
```suggestion
18. The next screen displays all the layers of your cluster profile. You need to apply your add-on profile to this cluster profile. Click on the **+** button above the cluster profile layers.
```

File: content/docs/06-integrations/00-prometheus-cluster-metrics.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-cluster-metrics.md:4]:
```suggestion
metaDescription: "Use the Prometheus Cluster Metrics addon pack to expose Palette resource metrics"
```

File: content/docs/06-integrations/00-prometheus-cluster-metrics.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-cluster-metrics.md:25]:
```suggestion
- CPU and Memory usage of the cluster and all the pods in the cluster.
```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:51]:
```suggestion
  - 20 GB Storage

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:59]:
```suggestion
    - 1 GB Storage

```

Comment (ritawatson) [reply content/docs/06-integrations/00-prometheus-operator.md:59]: Several of our docs say "GiB". We don't have a style for that yet and should probably come up with one since it's an inconsistency in our docs.

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:65]:
```suggestion
The Prometheus operator supports all the parameters exposed by the kube-prometheus-stack Helm Chart. Refer to the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) documentation for details.

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:166]:
```suggestion
You can configure the Prometheus server to accept metrics from Prometheus agents and become a centralized aggregation point for all Kubernetes metrics. Enabling this feature will expose port 9090 of the *prometheus-operator-prometheus* service. Use the generated service URL to provide other Kubernetes clusters with the installed [Prometheus Agent](/integrations/prometheus-agent) so that cluster metrics can be forwarded to the Prometheus server. 

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:171]:
```suggestion
Refer to the [Prometheus Remote Write Tuning](https://prometheus.io/docs/practices/remote_write/) resource to learn more about configuration options.

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:179]: @karl-cardenas-coding [Prometheus Cluster Metrics](/integrations/prometheus-cluster-metrics) seems to be a broken link.

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:189]:
```suggestion
* [Prometheus Alertmanager](https://www.prometheus.io/docs/alerting/latest/alertmanager/)

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:193]:
```suggestion
* Service monitors to scrape internal Kubernetes components

```

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (ritawatson) [inline content/docs/06-integrations/00-prometheus-operator.md:814]: @karl-cardenas-coding This seems to be a broken link.

**Review by ritawatson — APPROVED** [review]
Some suggestions. I couldn't find the Validation heading in the deploy-agent file to suggest adding a break between the heading and the first step. An extra line will help it not look cramped.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1239#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1239#pullrequestreview-1390435815" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1239#pullrequestreview-1388991679" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-prometheus-operator.md
Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-prometheus-operator.md:184]: Ah, the extra `/` at the end. Nice catch @ritawatson.

**Pull request merged**
